core/Tracker/Request.php

```diff
@@ -713,6 +714,18 @@ protected function getCookieDomain()
      */
     public function getVisitorId()
     {
+        try {
+            if ($this->getIdSiteUnverified()) {
+                $siteAttributes = Cache::getCacheWebsiteAttributes($this->getIdSite());
+            }
+        } catch (\Exception $e) {
+            // ignore any exception so method also works if site wasn't given or doesn't exist
+        }
+
+        if (isset($siteAttributes[PrivacyManager::TRACKER_CACHE_COOKIELESS_FORCED]) && $siteAttributes[PrivacyManager::TRACKER_CACHE_COOKIELESS_FORCED]) {
```

Comment:
A userId would be still accepted and I think a forced visitor Id as well. Only the third party cookie and the `_id` parameter which comes AFAIK from the cookie would be ignored

Reply:
makes sense. changed that